be/test/exprs/vectorized/math_functions_test.cpp

```diff
@@ -217,21 +217,35 @@ static void testRoundDecimal(const std::vector<std::string>& arg0_values, const
     } else if (c0_const && c1_const) {
         ASSERT_TRUE(res_column->is_constant());
         res_const = true;
-        decimal_res_column =
-                ColumnHelper::cast_to<TYPE_DECIMAL128>(FunctionHelper::get_data_column_of_const(res_column)).get();
+        auto maybe_nullable_res_column = FunctionHelper::get_data_column_of_const(res_column);
```

Comment:
Has relationship with this PR?

Reply:
No relation. Just add some tests for decimal truncate/round (don't want to open another pr 😵)

Reply:
better to create one new PR

Reply:
ok, I'll move tests to another pr